Wireframe/index.html

```diff
@@ -1,13 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <title>What is Git?</title>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="style.css">
```

I think this will help! 😉